README.md

```diff
@@ -110,6 +126,25 @@ None.
       roles:
         - role: geerlingguy.redis
 
+Setup replication
+
+    - hosts: redis_primary
+      gather_facts: true
+      become: true
+      roles:
+        - role: geerlingguy.redis
+          redis_replication: true
+          redis_bind_interface: "{{ ansible_eth0.ipv4.address }}"
+
+    - hosts: redis_secondary
+      gather_facts: true
+      become: true
+      roles:
+        - role: geerlingguy.redis
+          redis_replication: true
+          redis_slaveof: "{{ hostvars[groups['redis_primary'][0]].ansible_eth0.ipv4.address }} {{ redis_port }}"
+          redis_bind_interface: "{{ ansible_eth0.ipv4.address }}"
```

Comment:
What if a server/host/machine has an external interface, and an internal interface.
The user cannot specify which one to use, again would make sense to make this variable be set by the user.